pkg/configurator/methods_test.go

```diff
@@ -246,15 +246,15 @@ func TestCreateUpdateConfig(t *testing.T) {
 			name:                  "GetInitContainerImage",
 			initialMeshConfigData: &v1alpha1.MeshConfigSpec{},
 			checkCreate: func(assert *tassert.Assertions, cfg Configurator) {
-				assert.Equal("openservicemesh/init:v0.9.0", cfg.GetInitContainerImage())
+				assert.Equal("openservicemesh/init:v0.9.1-rc.1", cfg.GetInitContainerImage())
 			},
 			updatedMeshConfigData: &v1alpha1.MeshConfigSpec{
 				Sidecar: v1alpha1.SidecarSpec{
-					InitContainerImage: "openservicemesh/init:v0.8.2",
+					InitContainerImage: "openservicemesh/init:v0.9.0",
```

Comment:
Shouldn't this also be v0.9.1-rc.1 ?

Reply:
This value doesn't need to be updated each time since it doesn't relate to the current version at all.

I'd be in favor of (outside of this PR) changing this to something like `TEST` just to make it obvious that the exact value doesn't really matter as long as it's different from the default.

Reply:
This method is testing updates to the mesh config. I don't think the version number matters as long as it is different from the above version number (in this case v0.9.1-rc.1)